entries/h/home.saxo.json

```
@@ -0,0 +1,17 @@
+{
+    "SaxoTraderGO": {
+      "domain": "home.saxo",
```

It seems their web server isn't configured correctly for HTTPS redirections and therefore a `url` element is needed. 
```suggestion
      "domain": "home.saxo",
      "url": "https://www.home.saxo/",
```